### DISCUSSION TITLE
Enum.TryParse unexpected behavior

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#75] [General]
# Enum.TryParse unexpected behavior

Enums are very simple structures, but some functions like Enum.TryParse can have unexpected behavior. In this short blog post, we discover why and what are the alternatives.

https://steven-giesel.com/blogPost/64d479f1-823f-4fc7-a56a-2b1ff06dcf72

**Comment by ghadzhigeorgiev** [comment]
Wow, nice spot!
Thanks for pointing it!
